docs/getting_started/tutorial.md

```diff
@@ -953,14 +970,11 @@ you need to create a template `blog/template/blog/blog_tag_index_page.html`:
 {% endblock %}
 ```
 
-We're calling the built-in `latest_revision_created_at` field on the `Page`
-model - handy to know this is always available.
+In the preceding `blog_tag_index_page.html` template, you're calling the built-in `latest_revision_created_at` field on the `Page`model. It's handy to know this is always available.
 
-We haven't yet added an "author" field to our `BlogPage` model, nor do we have
-a Profile model for authors - we'll leave those as an exercise for the reader.
+There is currently no "author" field in your `BlogPage` model, nor do you have a `Profile` model for authors. Feel free to try this as an exercise on your own.
```

Comment:
This is pretty confusing since we do now have an `authors` field. Let’s remove this fully and also remove the corresponding template code:

```
              {% if blogpage.author %}
                <p>By {{ blogpage.author.profile }}</p>
              {% endif %}
```